dvc/commands/studio.py

```diff
@@ -48,12 +48,18 @@ def run(self):
 
         config_path = self.config.files["global"]
         ui.write(f"Authentication complete. Saved token to {config_path}.")
+        ui.write("Experiments are now being pushed automatically as git commits.")
```

Comment:
I guess we should make this conditional on whether there is an `origin` git remote.

Reply:
@skshetry Any ideas on how to do this?

Reply:
should we actually just try to push it and fail? (just mention that by default it will looking for an origin remote and default DVC remote(s)). Probably we could introduce an additional option to set a different Git remote to use.

Reply:
There is different remote for each branch. So we should not make this conditional during `studio login`.

Tbh, I don't think we need to display this message at all, as the only use of studio token is to push experiments.

Reply:
In that case, I think we need to handle the error better downstream during `exp run`. For example, here's what we show now:

```
$ dvc exp run -f
Reproducing experiment 'washy-doge'
Building workspace index                                      |6.00 [00:00, 1.64kentry/s]
Comparing indexes                                             |7.00 [00:00, 3.98kentry/s]
Applying changes                                               |0.00 [00:00,     ?file/s]
Running stage 'train':
> python src/stages/train.py --config=params.yaml
ERROR: 'origin' is not a valid Git remote or URL
```

Reply:
That is a change in behaviour for `EXP_AUTO_PUSH`.

@dberenbaum, since this is unrelated to the PR, do you mind if I work on that on a separate PR? 

I'll try to resolve https://github.com/iterative/dvc/pull/10331/files#r1510765500 within this PR though, and get rid of the UI message from this PR.